src/client/Microsoft.Identity.Client/Cache/Items/MsalAccountCacheItem.cs

```diff
@@ -48,6 +48,7 @@ internal MsalAccountCacheItem()
             string preferredCacheEnv,
             string clientInfo,
             string homeAccountId,
+            string accountSource,
```

Comment:
What will be the possible values of accountSource? Is this needed to be added to cache key?